CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 ## Unreleased
 
 ### Improvements
+
+- refactor(vpcCni): set node anti-affinity to not deploy to fargate
+  [#291](https://github.com/pulumi/pulumi-eks/pull/291)
+  - Note: this change expands on the divergence between the
+  [upstream CNI](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/config/v1.5/aws-k8s-cni.yaml),
+  Pulumi's [approach](https://github.com/pulumi/pulumi-eks/blob/master/nodejs/eks/cni/aws-k8s-cni.yaml),
+  and what AWS does by [default](https://github.com/aws/amazon-vpc-cni-k8s/issues/755) for Fargate clusters.
```

Comment:
I think we can leave this note out of the changelog - but perhaps include it somewhere in source - or in a readme next to our copy of this yaml.

Reply:
Done. Moved to `cni/README`.